src/components/Menu/MenuContent/MenuContent.tsx

```diff
@@ -35,16 +35,11 @@ export const MenuContent: FunctionComponent<MenuContentProps> = ({
 		return (
 			<div
 				className={classnames(className, 'c-menu__item')}
-				onClick={() => (onClick || (() => {}))(menuItemInfo.id)}
+				onClick={() => onClick(menuItemInfo.id)}
 				key={`menu-item-${menuItemInfo.id}`}
-				style={onClick ? { cursor: 'pointer' } : {}}
```

Comment:
waar heb je deze style naar toe gezet?

Reply:
in MenuContent.scss op de `c-menu__item` class